### PR TITLE
UI Updates

### PR DESCRIPTION
**Pull request by francisschmaltz**
Other UI changes made before 3.0 will be pushed here.

Currently:
- [x] Update license card BG
- [x] Update site-admin
- [x] Update user profile 

### License Card
New License Card BG:
screen shot 2019-01-10 at 1 31 47 pm
screen shot 2019-01-10 at 1 31 41 pm

Old License Card BG:
screen shot 2019-01-10 at 1 32 03 pm
screen shot 2019-01-10 at 1 32 08 pm

### Site Admin UI Updates
- The entire `explore` element is now a block link, not two separate buttons.
- `View all` buttons are listed on the far left to align
- Action buttons are now links

New Site Admin UI:
screen shot 2019-01-11 at 12 26 55 am

Old Site Admin UI:
screen shot 2019-01-11 at 12 30 01 am

### User Profile Updates
- Updated to display names on top instead of side to side
- Made `new organization` button take up full area of list element
- Added icon to `site admin` button in the sidebar
- Update subscriptions page to match site-admin style header

New UI:
screen shot 2019-01-11 at 4 13 45 pm
screen shot 2019-01-11 at 4 21 55 pm
screen shot 2019-01-11 at 4 29 41 pm

Old UI:
screen shot 2019-01-11 at 4 14 12 pm
screen shot 2019-01-11 at 4 22 11 pm
screen shot 2019-01-11 at 4 29 47 pm

### CODE REVIEW

File: web/src/components/Overview.scss [hunk 1]
```diff
@@ -16,6 +16,33 @@
...
+        position: absolute;
+        width: 3rem;
+        z-index: 9;
+        opacity: 0;
+        height: 2rem;
+        cursor: pointer;
+    }
+    .icon-rotate {
+        position: relative;
+        .icon-container {
```
Comment (felixfbecker) [inline web/src/components/Overview.scss]: Why do all of these selectors have to be child selectors? Please follow our [coding style guide](https://github.com/sourcegraph/sourcegraph/blob/master/doc/dev/web_app.md#styles), BEM and make sure your editor is configured to show stylelint warnings. Child selectors are unperformant, break encapsulation and are hard to maintain & debug.

File: web/src/components/Overview.scss [hunk 2]
```diff
@@ -48,6 +75,31 @@
...
+            color: $color-light-text-1;
+        }
+        &:hover {
+            text-decoration: none;
+            .overview-item__header {
+                text-decoration: underline;
+            }
+        }
+
+        a {
```
Comment (felixfbecker) [inline web/src/components/Overview.scss]: Why would should all `<a>` elements look different here? Should this maybe be a class instead?

File: web/src/enterprise/productSubscription/ProductCertificate.scss [hunk 3]
```diff
@@ -8,29 +8,40 @@
...
 
     &__logo {
         height: 5rem;
+        z-index: 9;
+    }
+    .btn {
+        z-index: 9;
+    }
+    .card-footer {
+        z-index: 9;
```
Comment (felixfbecker) [inline web/src/enterprise/productSubscription/ProductCertificate.scss]: These styles are duplicated, and again use child selectors. Address elements through descriptive BEM class names.

File: web/src/components/Overview.scss [hunk 4]
```diff
@@ -16,6 +16,33 @@
...
+        top: 0;
+        right: 0;
+        position: absolute;
+        width: 3rem;
+        z-index: 9;
+        opacity: 0;
+        height: 2rem;
+        cursor: pointer;
+    }
+    .icon-rotate {
```
Comment (felixfbecker) [inline web/src/components/Overview.scss]: This would be a modifier in BEM, i.e. `--rotated`.

Comment (francisschmaltz) [reply web/src/components/Overview.scss]: It's not a modifier (and this code was removed anyways). It's a description of a single element.

File: web/src/enterprise/productSubscription/ProductCertificate.scss (hunk 3 above)
Comment (francisschmaltz) [reply web/src/enterprise/productSubscription/ProductCertificate.scss]: https://github.com/sourcegraph/sourcegraph/issues/1836

**Pull request merged**